cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -74,7 +74,7 @@ akka.management {
 
       # If some discovered seed node will keep failing to connect for specified period of time,
       # it will initiate rediscovery again instead of keep trying.
-      probing-failure-timeout = 5 seconds
+      probing-failure-timeout = 3 seconds
```

Comment:
I think this needs some cleanup... The request coming back does not have a timeout it seems; yet this timeout is about "if we KEEP FAILING for this period, we'll resolve a new contact point and try again". I think we need to add a probe-timeout = 1.second perhaps and then should be good to have this at 3, so it's be 2/3 attempts after which we resolve again